tests/dataimportcron_test.go

```diff
@@ -447,7 +448,7 @@ var _ = Describe("DataImportCron", func() {
 		configureStorageProfileResultingFormat(format)
 
 		garbageSources := 3
-		for i := 0; i < garbageSources; i++ {
+		for i := 1; i <= garbageSources; i++ {
 			srcName := fmt.Sprintf("src-garbage-%d", i)
```

Comment:
Not sure I like this naming hack, oldDvName should have a better name than "src-garbage-0", e.g. "old-version-dv", and the loop can be reverted to 0-based.

Reply:
Sure, no problem. Fixed